editor/example/spec/islandviewer.ts

```diff
@@ -0,0 +1,276 @@
+import type { GoslingSpec } from '@gosling.schema';
+
+const circularRadius = 250;
+const centerRadius = 0.5;
+
+const linearHeight = 120;
+const linearSize = linearHeight / 6;
+const spec: GoslingSpec = {
+    title: 'IslandViewer 4 (Bertelli et al. 2017)',
+    subtitle: 'Salmonella enterica subsp. enterica serovar Typhi Ty2, complete genome.',
```

Comment:
I think it is helpful to have the reference link here.

```suggestion
    subtitle: 'Salmonella enterica subsp. enterica serovar Typhi Ty2, complete genome.',
      "description": "Reimplementation of https://www.pathogenomics.sfu.ca/islandviewer/accession/NC_004631.1/",
```

Reply:
When I apply this change I cannot see the description anywhere, where is it supposed to show up?

Reply:
It is just a note to the spec and is not supposed to show up somewhere.